src/lib/util/formatBankValue.ts

```diff
@@ -0,0 +1,22 @@
+import { toKMB } from 'oldschooljs/dist/util/util';
+
+const formatBankValue = (totalValue: number): string => {
+	if (totalValue < 1e9) {
```

Comment:
Use `const billion = 1_000_000_000;` instead (declared outside the function), for readability purposes